inst/examples/03-formats.Rmd

```diff
@@ -218,9 +218,195 @@ favicon: "favicon.ico"
+
+- Easy customisation of colours and fonts with
+  the bslib package (<https://pkgs.rstudio.com/bslib/>)
+
+- Built-in search (broken down by section) that helps readers quickly find t
+  they are looking for.
+
+- A sidebar containing a within-chapter table of contents that makes
+  navigation easy and helps provide context about your current position
+  within the chapter.
+
+- Thoughtful typography to make the contents as easy as possible to read,
+  regardless of the size of your device. A sticky header gets out of your
+  way when reading, but is easily accessible if you need it.
+
+- In-line footnotes mean you can read asides next to the next they refer
+  to. This theme is best paired with a reference style that generates
+  footnotes.
+
+- R syntax highlighting and autolinking by
+  the downlit package (<https://downlit.r-lib.org/>) is paired with a accessible
+  colour scheme designed by Alison Hill.
+  
+- Enhanced metadata for social sharing via platforms like Twitter, LinkedIn, and Facebook, so that each chapter shared will have a unique description, auto-generated based on that chapter's content.
+
+- Ability to configure links to a remote repository like GitHub or GitLab, allowing readers to easily view each chapter's source file or suggest an edit.
+
+#### Writing a `bs4_book`
+
+This theme is designed for books that use one chapter per page. 
+This means that each chapter is an `.Rmd` file, and each `.Rmd` file can contain one (and only one) chapter. 
+A chapter *must* start with a first-level heading: `# A good chapter`, and can contain one (and only one) first-level heading. 
+
+Use second-level and higher headings within chapters like: 
+
+```md
+#   A good chapter
+
+##  A short section
+
+### An even shorter section
+```
+
+The first- and second-level headings will appear in that chapter's sidebar on the upper right as a "sticky" within-chapter table of contents that stays in place as you scroll down. In the sidebar, when a section is navigated to, subheadings will auto-expand such that third-level headings like "An even shorter section" are visible.
+
+The `index.Rmd` file is required, and is also your first book chapter. It will be the homepage when you render the book.
+
+A YAML header in `index.Rmd` for a `bs4_book` would look like this:
+
+```yaml
+---
+title: "A Minimal Book Example"
+author: "John Doe"
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+output: bookdown::bs4_book
+url: https://bookdown.org/johndoe/bookdown-demo
+cover-image: cover.png
+description: |
+  This is a minimal example of using the bookdown package to write a book.
+  The output format for this example is bookdown::bs4_book.
+---
+```
+
+#### Callout blocks
+
+The `bs4_book` theme also includes special callout blocks, like this `.rmdnote`.
+
+::: {.rmdnote}
+You can use **markdown** inside a block.
+
+```{r collapse=TRUE}
+head(beaver1, n = 5)
+```
+
+:::
```

Comment:
You need verbatim here, otherwise it is process by Pandoc, and the bookdown's book style will apply for this blocks
![image](https://user-images.githubusercontent.com/6791940/127390170-002422d3-fee1-4898-a0cb-6aec0e8db49b.png)


`````suggestion
````markdown
::: {.rmdnote}
You can use **markdown** inside a block.

```{r collapse=TRUE}`r ''`
head(beaver1, n = 5)
```

:::
````
`````